src/globus_cli/commands/streams/tunnel/create.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+import uuid
+
+import click
+import globus_sdk
+from globus_sdk.services.transfer.data.tunnel_data import CreateTunnelData
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import OMITTABLE_INT, OMITTABLE_STRING, command
+from globus_cli.termio import display
+
+from ._common import TUNNEL_STANDARD_FIELDS
+
+
+@command("create", short_help="Create a Globus Tunnel.")
+@click.argument(
+    "initiator_stream_ap_id", metavar="INITIATOR_STREAM_AP_ID", type=click.UUID
+)
+@click.argument(
+    "listener_stream_ap_id", metavar="LISTENER_STREAM_AP_ID", type=click.UUID
+)
+@click.option(
+    "--label",
+    type=OMITTABLE_STRING,
+    default=globus_sdk.MISSING,
+    help="A friendly label to associate with the Tunnel.",
+)
+@click.option(
+    "--lifetime-minutes",
+    type=OMITTABLE_INT,
+    default=globus_sdk.MISSING,
+    help="The amount of time in minutes that the tunnel will exist.",
+)
+@click.option(
+    "--restartable",
+    is_flag=True,
+    help=(
+        "A boolean to indicate if the tunnel should be restart in the "
+        "event of an error."
+    ),
+)
+@LoginManager.requires_login("auth", "transfer")
+def create_tunnel_command(
+    login_manager: LoginManager,
+    *,
+    initiator_stream_ap_id: uuid.UUID,
+    listener_stream_ap_id: uuid.UUID,
+    label: str | globus_sdk.MissingType,
+    lifetime_minutes: int | globus_sdk.MissingType,
+    restartable: bool,
+) -> None:
+    """
+    Create a Globus Tunnel.
```

Comment:
Tunnel (as a name for the resource) appears in uppercase and lowercase in this PR. Recommending lowercase for consistency.

```suggestion
    Create a Globus tunnel.
```